spec/features/staff/users_can_create_a_budget_spec.rb

```diff
@@ -91,9 +88,6 @@
 
         click_button t("default.button.submit")
 
-        choose("budget[budget_type]", option: "1")
-        choose("budget[status]", option: "1")
-        choose("budget[funding_type]", option: "1")
```

Comment:
Reading the spec, it is designed to check that if you enter a value outside of the range you get the right error, the fact you also get errors for other values being incorrect isn't strictly speaking under test here i.e. it doesn't look like it would effect the outcome, but I think it makes sense that this spec only has the value incorrect and not any other attribute, and that means we need to set a financial year as well as keeping the funding_type, I'll go ahead and do that.